packages/api/src/jobs/rss/refreshAllFeeds.ts

```diff
@@ -137,8 +137,6 @@ export const queueRSSRefreshFeedJob = async (
   }
   return queue.add('refresh-feed', payload, {
     jobId: jobid,
-    removeOnComplete: true,
```

Comment:
I think we might need this for this job because of the jobid hack

Reply:
yeah, i moved this config into queue configuration